properties-panel-extension/README.md

```diff
@@ -2,17 +2,16 @@
 
 This example shows how to extend the [bpmn-js-properties-panel](https://github.com/bpmn-io/bpmn-js-properties-panel) with custom properties.
```

Comment:
We could add a note on top of the README that it works differently in `properties-panel@0.x` (and same on [properties-panel](https://github.com/bpmn-io/bpmn-js-properties-panel) README).

Reply:
https://github.com/bpmn-io/bpmn-js-examples/pull/144